content/articles/2019-09-pkgdown-1-4-0.Rmarkdown

```diff
@@ -0,0 +1,60 @@
+---
+title: pkgdown 1.4.0
+author: Hadley Wickham
+date: '2019-09-03'
+slug: pkgdown-1-4-0
+photo:
+  url: https://unsplash.com/photos/ytMmedc6y44
+  author: noodle kimm
+categories:
+  - package
+tags:
+  - pkgdown
+---
+
+We're happy to announce that [pkgdown](https://pkgdown.r-lib.org/) 1.4.0 is now
+available on CRAN. pkgdown is designed to make it quick and easy to build a
+website for your package. 
+
+## Development and deployment modes
+
+The main change in this version is that `build_site()`, `build_reference()` and  `build_home()` gain a `devel` parameter that controls whether you're in deployment or development mode. 
+
+*  `devel = FALSE` activates deployment mode, which installs the package into a 
+    temporary library, and runs examples/articles in a new process. It's the
+    default for `build_site()`. 
+  
+    If you're using `build_site()` as part of an existing deployment workflow, 
+    you may need to set `install = FALSE` so that the package doesn't get 
+    installed twice.
+
+*   `devel = TRUE` activates development mode, simulates package loading with
+    `pkgload::load_all()` and does all work in the same process. This makes it
+    most suitable for rapid iteration, and so is the default for 
+    `build_reference()`. 
+  
+There were two related changes to `build_reference()` and `build_home()`: `build_reference()` no longer runs `devtools::document()` and `build_home()` no longer re-builds `README.Rmd`.  This makes pkgdown's responsibilities more clear: it only modifies files in `doc/`.
```

Comment:
```suggestion
There were two related changes to `build_reference()` and `build_home()`: `build_reference()` no longer runs `devtools::document()`, and `build_home()` no longer re-builds `README.Rmd`.  This makes pkgdown's responsibilities clearer: it only modifies files in `doc/`.
```

Reply:
Is this a conscious use of hyphenated re-build to harken to the function names? If not, I think rebuild can be used in closed form.